source/modulo_core/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ find_package(ament_cmake_python REQUIRED)
 
 ament_auto_find_build_dependencies()
 
-find_package(control_libraries 6.0.7 REQUIRED COMPONENTS state_representation)
-find_package(clproto 6.0.0 REQUIRED)
+find_package(control_libraries 6.0.8 REQUIRED COMPONENTS state_representation)
```

Comment:
Since you already changed the required CL version here you might as well update it to 6.1 now following the latest release!